apps/external-devices/device-descriptors/Nintendo_WiiMote.json

```diff
@@ -0,0 +1,16 @@
+{
+    "deviceVendor": "Nintendo",
+    "deviceName": "WiiMote",
```

Comment:
Wii Remote, technically - and it also includes support for the Nunchuk, Wii Classic Controller, and Wii Balance Board (which might be separate descriptors) as well as the Motion+ add-on